client/src/Components/Login.js

```diff
@@ -59,6 +60,9 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
+const clientId =
+  "241513642950-4aom6i0o8dglg9j8cdrp4vgvgrp1d4hn.apps.googleusercontent.com";
```

Comment:
Should be read from env file and not hardcoded. Please create a write up for future devs on how to set this up.